prime-angular-frontend/src/app/modules/site-registration/pages/business-licence/business-licence.component.ts

```diff
@@ -30,13 +32,16 @@ export class BusinessLicenceComponent implements OnInit {
   public form: FormGroup;
   public title: string;
   public routeUtils: RouteUtils;
+  public businessLicence: BusinessLicence;
   public businessLicenceDocuments: BusinessLicenceDocument[];
   public uploadedFile: boolean;
   public hasNoBusinessLicenceError: boolean;
   public doingBusinessAsNames: string[];
   public isCompleted: boolean;
   public SiteRoutes = SiteRoutes;
 
+  public deferredLicence: boolean;
```

Comment:
Looks like this follows the value of the slide toggle.  You can get a reference to the slide toggle using `@ViewChild` and then directly access the source of truth for its current value instead of using a member variable to relay the value.